unstable/wlr-window-list.xml

```diff
@@ -0,0 +1,148 @@
+
+      Each window has a list of outputs it is visible on, conveyed to the client
+      with the enter_output and leave_output events.
+    </description>
+
+    <event name="title">
+      <description summary="title change">
+        This event is emitted whenever the title of the window changes
+        and immediately after the creation of the zwlr_window_v1
+      </description>
+      <arg name="title" type="string"/>
+    </event>
+
+    <event name="app_id">
+      <description summary="app-id change">
+        This event is emitted whenever the app-id of the window changes
+        or after it is created (the initial title)
+      </description>
+      <arg name="app_id" type="string"/>
+    </event>
+
+    <event name="enter_output">
+      <description summary="window entered an output">
+        This event is emitted whenever the window becomes visible on
+        the given output. A window may be visible on multiple outputs.
+      </description>
+      <arg name="output" type="object" interface="wl_output"/>
+    </event>
+
+    <event name="leave_output">
+      <description summary="window left an output">
+        This event is emitted whenever the window stops being visible on
+        the given output. It is guaranteed that an entered-output event
+        with the same output has been emitted before this event.
+      </description>
+      <arg name="output" type="object" interface="wl_output"/>
+    </event>
+
+    <event name="virtual_desktop">
+      <description summary="window was moved to a virtual desktop">
+        This event is emitted with the initial virtual desktop the window
+        is on and whenever it is moved to a different virtual desktop later.
+      </description>
+      <arg name="desktop" type="string"/>
+    </event>
+
+    <event name="maximized">
+      <description summary="window's maximized state changed"/>
+      <arg name="state" type="uint"/>
+    </event>
+
+    <request name="set_maximized">
+      <description summary="requests a change in the window maximized state">
+        Requests that the window maximized state be changed.
+        No guarantee the window will actually be (un)maximized.
+      </description>
+      <arg name="maximized" type="uint"/>
+    </request>
+
+    <event name="minimized">
+      <description summary="window's minimized state changed"/>
+      <arg name="state" type="uint"/>
+    </event>
+
+    <request name="set_minimized">
+      <description summary="requests a change in the window minimized state">
+        Requests that the window minimized state be changed.
+        No guarantee the window will actually be (un)minimized.
+      </description>
+      <arg name="minimized" type="uint"/>
+    </request>
+
+    <request name="focus">
+      <description summary="request focus">
+        Request that this window is focused. However, there is no guarantee
+        the window will be actually focused.
+      </description>
+    </request>
```

Comment:
Should we call this "activate"? Also, there's no event which tells you if the window is focused or not.

Reply:
hmm, yes, this will better fit with xdg-shell wording